### PR TITLE
Clean relayer v1

### PR DESCRIPTION
**Pull request by 1devNdogs**
Latest css clean.

- Removed unused bootstrap components.
- Fix several typos and styles.
- Added subspace favicons
- Added latest subspace design styles.

- Also added a [Relayer Context](https://github.com/subspace/subspace-relayer-front-end/blob/3b0fe99614d8fbae30a0372713044b5cc3cd7843/src/context/RelayerContext.tsx) for FeedTotals on every [parachain row.
](https://github.com/subspace/subspace-relayer-front-end/blob/3b0fe99614d8fbae30a0372713044b5cc3cd7843/src/components/ParachainRow.tsx)

I tested with the latest [subspace changes](https://github.com/subspace/subspace/pull/64) 

image

### CODE REVIEW

File: src/components/utils.ts [hunk 1]
```diff
@@ -0,0 +1,11 @@
+export const bytesToSize = (bytes: number): string => {
+  if(bytes === 0) return '0 B';
+  const sizes = ["Bytes", "KB", "MB", "GB", "TB"];
+  const i = parseFloat(Math.floor(Math.log(bytes) / Math.log(1024)).toString());
```
Comment (nazar-pc) [inline src/components/utils.ts:4]: This is a cumbersome way to do it. Why would you want to convert integer to string and then back to float, which is not really a float, but a `number` in JS anyway?

File: src/config/interfaces/Parachain.ts [hunk 2]
```diff
@@ -14,3 +16,7 @@ export interface ParachainProps {
   explorer: string;
 }
 
+export interface Totals extends Struct {
+  readonly size_: u64;
```
Comment (nazar-pc) [inline src/config/interfaces/Parachain.ts:20]: Weird, do you know why does it have `_` at the end?

File: src/context/RelayerContext.tsx [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+import { ApiPromiseContext, SystemContext } from "context";
+import { ApiPromise } from "@polkadot/api";
+import { useProvider } from "./ProviderContext";
+
+async function getFeedTotals(api: ApiPromise, n?: number): Promise<Totals[]> {
+  const feedsTotals: Array<Totals> = Array<Totals>();
+  for (const { feedId } of parachains) {
+    const totals = await api.query.feeds.totals(feedId);
+    const _totals: Totals = api.registry.createType("Totals", totals);
+    feedsTotals.push(_totals);
```
Comment (nazar-pc) [inline src/context/RelayerContext.tsx:19]:
```suggestion
    feedsTotals.push(api.registry.createType("Totals", totals));
```

File: src/components/ParachainTable.tsx [hunk 4]
```diff
@@ -42,18 +31,20 @@ const ParachainTable = () => {
               const data_: any = api.registry
                 .createType("Bytes", args[1])
                 .toHuman();
+              //  console.log(data_);
```
Comment (isSerge) [inline src/components/ParachainTable.tsx:34]: could you clean up please?

File: src/components/ParachainTable.tsx [hunk 5]
```diff
@@ -42,18 +31,20 @@ const ParachainTable = () => {
               const data_: any = api.registry
```
Comment (isSerge) [inline src/components/ParachainTable.tsx:31]: why _ at the end of the variable name?

File: src/components/ParachainTable.tsx [hunk 6]
```diff
@@ -64,29 +55,13 @@ const ParachainTable = () => {
...
-        setMoreStorage(newMoreStorage);
-        setMoreBlocks(newMoreBlocks);
       })
       .catch((e) => console.error(e));
   }, [api, isApiReady]);
 
   return (
     <div>
-      <Header acumulatedBytes={moreStorage} totalBlocks={moreBlocks}></Header>
+      <Header></Header>
```
Comment (isSerge) [inline src/components/ParachainTable.tsx:64]: `<Header />`?

File: src/layout/MainLayout.tsx [hunk 7]
```diff
@@ -1,12 +1,19 @@
...
+import { ApiPromiseContextProvider, RelayerContextProvider } from "context";
 
 const MainLayout = () => {
   return (
     <>
       <NavBar></NavBar>
-      <ParachainTable />
+      
+      <ApiPromiseContextProvider>
+        <RelayerContextProvider>
```
Comment (isSerge) [inline src/layout/MainLayout.tsx:12]: Do we use `RelayerContextProvider` in any child component?

File: src/components/utils.ts (hunk 1 above)
Comment (1devNdogs) [reply src/components/utils.ts]: Was totally unnecesary, have removed the parseFloat and toString.

File: src/components/utils.ts [hunk 8]
```diff
@@ -1,7 +1,7 @@
 export const bytesToSize = (bytes: number): string => {
   if(bytes === 0) return '0 B';
   const sizes = ["Bytes", "KB", "MB", "GB", "TB"];
```
Comment (nazar-pc) [inline src/components/utils.ts:3]: Just a nitpick, but I would to this instead to be precise:
```suggestion
  const sizes = ["Bytes", "KiB", "MiB", "GiB", "TiB"];
```

**Pull request merged**